src/app/version-stamp.component.js

```diff
@@ -0,0 +1,43 @@
+export const versionStampComponent = {
+  selector: 'appVersionStamp',
+  template: `
+    <style>
+      .version-container {
+        color: gray;
+        display: flex;
+        flex-flow: row wrap;
+        justify-content: space-between;
+        margin: 8px;
+        padding: 8px;
+      }
+      .version {
+        font-size: 0.75em;
+        font-style: italic;
+      }
+      .version img {
+        vertical-align: top;
+        height: 24px;
+      }
+      #version-stamp-divider {
+        margin-top: 64px;
+      }
+    </style>
+
+    <md-divider id="version-stamp-divider"></md-divider>
+    <div class="version-container">
+      <span class="version"><img src="{{ $ctrl.angularJsIcon }}" alt="" height="24"> AngularJS v{{ $ctrl.versions.angularjs.full }} ({{ $ctrl.versions.angularjs.codeName }})</span>
+      <span class="version"><img src="{{ $ctrl.angularjsMaterialIcon }}" alt="" height="24"> AngularJS Material v{{ $ctrl.versions.md }}</span>
```

Comment:
We are setting the height in CSS, so we don't need it here:

```suggestion
      <span class="version"><img src="{{ $ctrl.angularJsIcon }}" alt=""> AngularJS v{{ $ctrl.versions.angularjs.full }} ({{ $ctrl.versions.angularjs.codeName }})</span>
      <span class="version"><img src="{{ $ctrl.angularjsMaterialIcon }}" alt=""> AngularJS Material v{{ $ctrl.versions.md }}</span>
```